cypress/integration/group3/mytools.ts

```diff
@@ -14,7 +14,24 @@
  *     limitations under the License.
  */
 import { DockstoreTool } from '../../../src/app/shared/openapi';
-import { goToTab, resetDB, setTokenUserViewPort, typeInInput } from '../../support/commands';
+import { goToTab, resetDB, setTokenUserViewPort, setTokenUserViewPortCurator, typeInInput } from '../../support/commands';
+
+describe('Should handle no tools correctly', () => {
+  resetDB();
+  setTokenUserViewPortCurator(); // Curator has no tools
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: 'GET',
+      url: /github.com\/organizations/,
+      response: ['dockstore'],
+    });
+  });
+  it('My tools should prompt to register a tool', () => {
+    cy.visit('/my-tools');
+    cy.contains('Add Tool');
```

Comment:
Apologies in advance for being borderline nitpicky, and I vaguely recall having this convo before, but if not:
We use 'Register Workflow' in one place, and 'Add Tool' here.  For the sake of consistency, maybe we pick one verb?

Reply:
I noticed the difference for the first time in doing this PR, debated whether to change it, and had decided to leave it, just to keep the PR focused. But looking again, the mocks for the new screen use "Register", so I'll go with that.

Reply:
On further reflection, I'm going to leave it. It has a small ripple effect where I don't want risk introducing something.